alphadia/outputtransform.py

```diff
@@ -1024,6 +1025,30 @@ def _build_run_stat_df(
         else:
             logger.warning(f"Error reading calibration manager for {raw_name}")
 
+        if os.path.exists(
+            raw_file_manager_path := os.path.join(
+                folder, peptidecentric.PeptideCentricWorkflow.RAW_FILE_MANAGER_PKL_NAME
+            )
+        ):
+            raw_file_manager = RawFileManager(
+                path=raw_file_manager_path, load_from_file=True
+            )
+            prefix = "raw."
+            stats = raw_file_manager.stats
+
+            # deliberately mapping explicitly to avoid coupling stats to the output too tightly
+            base_dict[f"{prefix}rt_limit_min"] = stats["rt_limit_min"]
+            base_dict[f"{prefix}rt_limit_max"] = stats["rt_limit_max"]
+
+            base_dict[f"{prefix}rt_duration_sec"] = stats["rt_duration_sec"]
```

Comment:
I actually agree with Feng that we should convert the values to minutes. We can store them as minutes from the beginning.